libs/application/template-api-modules/src/lib/modules/template-api.module.ts

```diff
@@ -1,18 +1,24 @@
 import { DynamicModule } from '@nestjs/common'
 
 // Imports of custom template API modules
-import { modules } from './templates'
+import { modules, clientModules, clientConfigs } from './templates'
 
 import { BaseTemplateAPIModuleConfig } from '../types'
 
 import { TemplateAPIService } from './template-api.service'
+import { ConfigModule } from '@island.is/nest/config'
 
 export class TemplateAPIModule {
   static register(config: BaseTemplateAPIModuleConfig): DynamicModule {
     return {
       module: TemplateAPIModule,
       imports: [
+        ...Object.values(clientModules).map((Module) => Module),
```

Comment:
`clientModules` is an array of the modules right, can do this: `...clientModules,`?

Reply:
Also, don't need to import the client modules here, just load their configs.